fuzz_lightyear/fuzzer.py

```diff
@@ -102,10 +103,14 @@ def _fuzz_string(
 
     # TODO: Handle a bunch of swagger string formats.
     # https://swagger.io/docs/specification/data-models/data-types/#string
+    kwargs = {}                                     # type: Dict[str, Any]
+
     if parameter.get('required', required):
-        return st.text(min_size=1)
+        kwargs['min_size'] = 1
+    if not get_settings().unicode_enabled:
+        kwargs['alphabet'] = string.ascii_letters
```

Comment:
Do we want to include numbers too? Or punctuation? Whitespace?

Reply:
D'oh. I'll change to `string.printable`.